app/server/api-generator/src/main/java/io/syndesis/server/api/generator/soap/parser/BindingHelper.java

```diff
@@ -426,7 +450,7 @@ protected XmlSchema createSoapSchema(SchemaCollection targetSchemas) {
             final XmlSchemaAnnotated annotated = part.getXmlSchema() != null ? part.getXmlSchema() :
                 schemaCollection.getTypeByQName(part.getTypeQName());
 
-            final QName name = part.getConcreteName();
```

Comment:
Are you sure we don't need a check to decide which one to use? Maybe if one is not set, use the other? I can't remember why I chose concrete name back then.

Reply:
> LGTM - nothing jumps out from AtlasMap perspective. Any specific concern I have to look deeper, e.g. possibility to change at AtlasMap side?

Not sure if the XML namespace needs to be included by default in AtlasMap, that way it can be omitted from here. It's added for all cases now.

Reply:
There is virtually no documentation on the difference between these, tests seem to confirm that `getName` is good enough.